hazelcast/src/main/java/com/hazelcast/query/impl/getters/AbstractMultiValueGetter.java

```diff
@@ -231,4 +232,11 @@ static void validateModifier(String modifier) {
         parseModifier(modifier);
     }
 
+    protected static String composeAttributeValueExtractionFailedMessage(Member member) {
+        return "Attribute value extraction failed for: " + member + ". Make sure attribute values or collection/array " +
```

Comment:
maybe we can decrease line lengths to something around 80 characters to make it more human readable.

Reply:
sorry, i meant formatting of the code to read from java file.